openy.packages.yml

```diff
@@ -0,0 +1,171 @@
+  help: '<p>Extended set of editorial breaks: layouts, blocks, paragraphs.</p>
+         <iframe width="560" height="315"
+             src="https://www.youtube.com/embed/kpUqb1WPXJM"
+             frameborder="0" allow="autoplay; encrypted-media"
+             allowfullscreen></iframe>'
+  description: '...'
+  modules:
+    - openy_block_date
+    - openy_block_featured_highlights
+    - openy_prgf_block_date
+    - openy_prgf_featured_highlights
+
+membership:
+  name: 'Membership'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/qTrGaisDWP0" frameborder="0"
+      allow="autoplay; encrypted-media" allowfullscreen></iframe>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+    - openy_node_mbrshp
+    - openy_calc
+    - openy_prgf_mbrshp_calc
+
+social_sharing:
+  name: 'Social Sharing'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: ''
+  modules:
+    - openy_addthis
+
+alerts:
+  name: 'Alerts'
+  help: '<dl>
+            <dt>Definition list</dt>
+            <dd>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+         aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+         commodo consequat.</dd>
+            <dt>Lorem ipsum dolor sit amet</dt>
+            <dd>Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+         aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+         commodo consequat.</dd>
+         </dl>'
+  description: '...'
+  modules:
+    - openy_node_alert
+
+analytics:
+  name: 'Analytics'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum
+  tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper.
+  Aenean ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+    - google_tag
+    - google_analytics
+
+embeddable_header_footer:
+  name: 'Embeddable header & footer'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>'
```

Comment:
instead of adding placeholder text, same some time for development and add true help )

Reply:
all descriptions will be replaced later in one scope by true descriptions.